packages/animations/browser/src/render/shared.ts

```diff
@@ -183,7 +183,15 @@ if (_isNode || typeof Element !== 'undefined') {
   _query = (element: any, selector: string, multi: boolean): any[] => {
     let results: any[] = [];
     if (multi) {
-      results.push(...element.querySelectorAll(selector));
+      // DO NOT REFACTOR TO USE SPREAD SYNTAX.
+      // For element queries that return sufficiently large NodeList objects,
+      // using spread syntax to populate the results array causes a RangeError
+      // due to the call stack limit being reached.
+      // See following issue: #38551
```

Comment:
Great suggestion to improve the comment @AleksanderBodurri 👍 Thanks @basherr for updating it.

I'd propose to capture a comment from Joost that NodeList is not iterable in IE11, my proposal:

```suggestion
      // DO NOT REFACTOR.
      // For element queries that return sufficiently large NodeList objects,
      // using spread syntax to populate the results array causes a RangeError
      // due to the call stack limit being reached. `Array.from` can not be used
      // as well, since NodeList is not iterable in IE 11, see
      // https://developer.mozilla.org/en-US/docs/Web/API/NodeList
      // More info is available in #38551.
```

Reply:
sounds good!